src/ai/whylabs/templates/batch_bigquery_template.py

```diff
@@ -147,6 +147,11 @@ def process_batch(self, batch: List[Dict[str, Any]]) -> Iterator[Tuple[SegmentDe
             if len(dataframe) == 0:
                 continue
 
+            if len(self.segment_columns_list) > 1:
+                for col in self.segment_columns_list:
+                    if dataframe[col].isna().values.any():
+                        self.logger.warning(f"Segmenting on column {col} with null values leads to a whylogs KeyError")
+
```

Comment:
Lets just make it raise if there is going to be an exception.

Also, we can move this up above the for loop and do it once.

Reply:
done